nifi-toolkit/nifi-toolkit-admin/src/test/groovy/org/apache/nifi/toolkit/admin/client/NiFiClientFactorySpec.groovy

```diff
@@ -121,21 +122,86 @@ class NiFiClientFactorySpec extends Specification {
 
     }
 
+    def "should verify wildcard in CN in certificate based on subjectDN"(){
+
+        given:
+        final String EXPECTED_DN = "CN=*.nifi.apache.org,OU=Security,O=Apache,ST=CA,C=US"
+        Certificate[] certificateChain = generateCertificateChain(EXPECTED_DN,ISSUER_DN)
+        def mockSession = Mock(SSLSession)
+        DefaultHostnameVerifier verifier = new DefaultHostnameVerifier()
+        mockSession.getPeerCertificates() >> certificateChain
+
+        when:
+        def verified = verifier.verify("client.nifi.apache.org",mockSession)
```

Comment:
Please rename these variables (and in following tests) to be more descriptive, i.e. `def validSubdomainIsVerified`, `def validSubdomainIsVerified2`, `def invalidSubdomainIsNotVerified`.